model/src/main/java/org/projectnessie/model/IcebergTable.java

```diff
@@ -69,20 +63,38 @@
   @NotBlank
   public abstract String getMetadataLocation();
 
-  /** Opaque representation of Iceberg's {@code TableIdGenerators}. */
-  public abstract String getIdGenerators();
+  public abstract long getSnapshotId();
```

Comment:
do these need default values? particularly the shcema/specid etc? Or do they have to be specified every time

Reply:
Don't think those should have default values. It uses what Iceberg's `TableMetadata` says and not rely on a default value.